ports/atmel-samd/boards/itsybitsy_m4_express/mpconfigboard.mk

```diff
@@ -15,3 +15,4 @@ LONGINT_IMPL = MPZ
 CIRCUITPY_AUDIOBUSIO = 0
 
 CIRCUITPY_BITBANG_APA102 = 1
+OPTIMIZATION_LEVEL = 2
```

Comment:
Instead of adding this flag for individual boards, I think it would make sense to do -O2 by default for all SAMD51 and nRF52 Express boards. There may be space problems on non-Express boards.

Reply:
@dhalbert See comment below. I think we will need to implement this on a board-by-board basis.

Reply:
I don't think we can always use INTERNAL_FLASH_FILESYSTEM = 1 as a guide. There is at least one board (sam32) that has that set but has 1M of flash (i.e. plenty for the 10% size increase). And, future chips may well fall into that same category. I would consider it safer to have a separate flag.